packages/react/src/components/SplitButton/SplitButton.tsx

```diff
@@ -202,9 +206,10 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
         )}
         {Button.create(toggleButton, {
           defaultProps: () => ({
+            styles: styles.toggleButton,
             className: SplitButton.slotClassNames.toggleButton,
             disabled,
-            icon: 'stardust-arrow-down',
+            icon: 'chevron-down-medium',
```

Comment:
I think you should use `stardust-menu-arrow-down`

![image](https://user-images.githubusercontent.com/828692/70473407-daad8980-1a85-11ea-9161-dd2d6ed80e20.png)